src/source/authentication.rst

```diff
@@ -40,6 +40,11 @@ OAuth 2.0 defines the following roles of users and applications:
   In this context *authorization server* is the server hosting authentication
```

Comment:
shoud this be the 'authorization and token endpoints'? as authentication happens @ the tenant

Reply:
can you verify whether this is indeed correct or not?

Reply:
corrected

Reply:
good catch